packages/web/src/features/register-club/components/activity-report/ActivityReportFrame.tsx

```diff
@@ -40,54 +40,51 @@ const ActivityReportFrame: React.FC<ActivityReportFrameProps> = ({
   clubId,
 }) => {
   const { profile } = useAuth();
-  const { data, isLoading, isError, refetch } =
-    useGetActivityReportsForPromotional({
-      clubId,
-    });
+  const { data, isLoading, isError } = useGetActivityReportsForPromotional({
+    clubId,
+  });
 
   const openCreateActivityReportModal = () => {
     overlay.open(({ isOpen, close }) => (
       <CreateActivityReportModal
         clubId={clubId}
         isOpen={isOpen}
         close={close}
-        refetch={refetch}
       />
     ));
   };
 
   return (
-    <AsyncBoundary isLoading={isLoading} isError={isError}>
-      <FlexWrapper direction="column" gap={40}>
-        <SectionTitle>가등록 / 등록 취소 기간 활동 보고서</SectionTitle>
-        <StyledCard outline gap={32}>
-          <OptionOuter>
-            <Typography
-              fs={14}
-              fw="REGULAR"
-              lh={20}
-              color="GRAY.300"
-              ff="PRETENDARD"
-            >
-              활동 보고서는 최대 20개까지 작성 가능합니다
-            </Typography>
-            <IconButton
-              type="default"
-              icon="add"
-              onClick={openCreateActivityReportModal}
-            >
-              활동 보고서 작성
-            </IconButton>
-          </OptionOuter>
+    <FlexWrapper direction="column" gap={40}>
+      <SectionTitle>가등록 / 등록 취소 기간 활동 보고서</SectionTitle>
+      <StyledCard outline gap={32}>
+        <OptionOuter>
+          <Typography
+            fs={14}
+            fw="REGULAR"
+            lh={20}
+            color="GRAY.300"
+            ff="PRETENDARD"
+          >
```

Comment:
```suggestion
          <Typography
            fs={14}
            lh={20}
            color="GRAY.300"
          >
```
기본 prop은 지워도 될 것 같아요